packages/api/tests/lambdas/create-reconciliation-report.js

```diff
@@ -101,27 +102,25 @@ function storeCollectionsToDynamoDb(tableName, collections) {
   return storeToDynamoDb(tableName, putRequests);
 }
 
-async function fetchCompletedReport(Bucket, stackName) {
-  const Prefix = `${stackName}/reconciliation-reports/`;
-
-  const report = await awsServices.s3().listObjectsV2({ Bucket, Prefix }).promise()
-    .then((response) => response.Contents[0].Key)
-    .then((Key) => awsServices.s3().getObject({ Bucket, Key }).promise())
-    .then((response) => response.Body.toString())
-    .then(JSON.parse);
-
-  if (report.status === 'RUNNING') {
+async function fetchCompletedReport(t) {
+  const reportRecord = await new models.ReconciliationReport().scan();
```

Comment:
We shouldn't need to do a scan here, we should be able to do a get by report name

Reply:
I don't have a report name.

Reply:
Ok.  Updated the lambda to return the report record